webserver/views/user.py

```diff
@@ -98,6 +103,31 @@ def import_data():
                            lastfm_username=lastfm_username)
 
 
+@user_bp.route("/export")
+@login_required
+def export_data():
+    filename = request.args.get("filename")
+    if filename:
+        filename += ".json"
+        filename = secure_filename(filename)
+        output = {}
+        cassandra = webserver.create_cassandra()
+        user_id = current_user.musicbrainz_id
```

Comment:
`current_user.musicbrainz_id` can be passed directly to `fetch_listens` function call. No need to put it in a variable.